### PR TITLE
add first functionality

### PR DESCRIPTION
**Pull request by mishanya1111**

### CODE REVIEW

File: frontend/my-app/src/components/Root/Navbar.js [hunk 1]
```diff
@@ -10,9 +10,6 @@ export default function Navbar({ onLanguageToggle, onThemeToggle, userName }) {
                     <h1 className="title">Meet Up!</h1>
                 </NavLink>
                 <div className="controls">
-                    <button onClick={onLanguageToggle} className="control-button">
-                        Change Language
```
Comment (ShadowFlyX) [inline frontend/my-app/src/components/Root/Navbar.js:14]: Why deleted?

File: frontend/my-app/src/components/home/MeetupCard.js [hunk 2]
```diff
@@ -1,9 +1,13 @@
...
+
+export default function MeetupCard({ title, description, image, to }) {
     return (
-        <div className="meetup-card">
-            <img className="image" src={image} alt="Meetup Image" />
-            <h3 className="title">{title}</h3>
-            <p className="description">{description}</p>
-        </div>
+        <NavLink to={to}>
+            <div className="meetup-card">
```
Comment (ShadowFlyX) [inline frontend/my-app/src/components/home/MeetupCard.js:6]: Why?

File: frontend/my-app/src/components/Root/Navbar.js (hunk 1 above)
Comment (mishanya1111) [reply frontend/my-app/src/components/Root/Navbar.js:14]: we add this late

File: frontend/my-app/src/components/home/MeetupCard.js (hunk 2 above)
Comment (mishanya1111) [reply frontend/my-app/src/components/home/MeetupCard.js:6]: because

**Comment by ShadowFlyX** [comment]
All seems good

**Pull request merged**